enterprise/models.py

```diff
@@ -6,11 +6,14 @@
 
 from uuid import uuid4
 
-from simple_history.models import HistoricalRecords  # likely a bug in import order checker
+import os
+from django.conf import settings
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords  # likely a bug in import order checker
```

Comment:
We'll need to keep an eye on this -- for some reason @e-kolpakov had to keep this import statement at the top of the list.